apps/group/app/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Group",
```

Comment:
This file is a duplicate of the `manifest.json` in the app root. They serve the same purpose. Also, `short_name` should be the same as `name`, as we only use `name`.

The icon at `icons[0]` will actually be used for the icon in the sidebar as well, just a small side note.

We also won't use `start_url`, `display`, `theme_color` or `background_color` - so these are just no-ops 😊

Reply:
Cool, I’ll remove it!